.appveyor.yml

```diff
@@ -1,26 +1,54 @@
-version: 1.0.{build}
-
-image: Visual Studio 2015
-platform: x64
-
-environment:
-  matrix:
-  - COMPILER: msys2
-    PLATFORM: x64
-    MSYS2_ARCH: x86_64
-    MSYS2_DIR: msys64
-    MSYSTEM: MINGW64
-    BIT: 64
-
-build_script:
-  - '%APPVEYOR_BUILD_FOLDER%\win_build.bat'
-
-test_script:
-  - '%APPVEYOR_BUILD_FOLDER%\test\test_libpostal.exe'
-
-after_build:
-  - 7z a libpostal.zip %APPVEYOR_BUILD_FOLDER%\libpostal.dll %APPVEYOR_BUILD_FOLDER%\libpostal.def %APPVEYOR_BUILD_FOLDER%\libpostal.exp %APPVEYOR_BUILD_FOLDER%\libpostal.lib %APPVEYOR_BUILD_FOLDER%\src\libpostal.h
-
-artifacts:
-  - path: libpostal.zip
-    name: libpostal
+version: 1.0.{build}
+image: Visual Studio 2015
+
```

Comment:
adding

```yaml
branches:
  only:
    - master
```

makes sure only one build is kicked off on feature branches.

Reply:
Added to new commit

Reply:
I've noticed since adding this I am no longer able to test changes using the CI within branches on my fork.  Would you mind testing adding `skip_branch_with_pr: true` to the `.appveyor.yml` config within this Pull request? It seems it may not have worked when changed within the UI as the .yml file overrides the UI settings.